python/pyspark/sql/tests.py

```diff
@@ -56,6 +56,15 @@
 from pyspark.sql.utils import AnalysisException, ParseException, IllegalArgumentException
 
 
+_have_arrow = False
+try:
+    import pyarrow
+    _have_arrow = True
```

Comment:
Do you mean to automatically enable the Arrow functionality if pyarrow installed?  Right now it is enabled manually with a flag `useArrow` in the public API `toPandas`.  If enabled and pyarrow is not installed, it will give an import error.